### ISSUE TITLE
Cyclic Wireless Transmitter next to RedstoneTunnel causes NullPointerException

### ISSUE THREAD

**Issue by Frikkle** [CompactMods/CompactMachines#390]
**Compact Machines version**: 3.0.17
**Forge version**: forge-14.23.5.2781
**I am not using Optifine**: Yes, I am not using Optifine
**Link to Crashlog**: [here](https://pastebin.com/hcAbdY0T)

**Description of the problem including expected versus actual behavior:**
When inside a Compact Machine (5x5x5 in this case); having a Redstone Tunnel set to input mode (direction: top) and placing a Wireless Transmitter (from Cyclic) directly against it causes the client to crash and brick it. Every reconnect force quits the client afterwards when joining the server.

When connecting the Wireless  transmitter and the Redstone Tunnel via an intermediate piece of Redstone Dust in between, all works fine. My attempt at compacting things resulted in the crash and brick of my client.

Expected behaviour: since it is known, mentioned and document that Tile Entities are not properly supported when transferring things from and to the CM I would have expected this not to work. I would however, expect the client not to crash and the NullPointerException to be handled.

**Steps to reproduce:**
1. Create a 5x5x5 CM.
2. Put a Lever on top.
3. Place a Redstone Tunnel on a side, set it to input mode and direction: top.
4. Place a Cyclic Wireless Receiver in the CM.
5. Link a Cyclic Wireless Transmitter to it by punching the Receive with the Transmitter.
6. Place the Wireless Transmitter directly against the Redstone Tunnel.
7. Client crash.

**Comment by Frikkle** [comment]
Cannot reproduce in SP-setup; can reproduce on MP-setup. Not sure what's causing it (game-wise).

Code-wise I cloned the code and added some additional null-checks in the code that throws the error for me. I structured the code changes to be in line with the current code-styling and structure so it's easy to review and patch in. About to create a PR for it.

EDIT: oh, and not unimportant: the newly compiled version with changes resolves all the issues we were experiencing.
